projects/storefrontstyles/scss/layout/page-templates/_landing-page-2.scss

```diff
@@ -5,6 +5,15 @@
   margin: auto;
 
   cx-page-slot {
+    cx-banner {
+      cx-media {
+        img {
+          display: flex;
```

Comment:
I'd rather keep a generic style rule set for all media in Spartacus. Can we move/adapt the rules in `_media.scss` instead?